src/core/server/config/__tests__/config_service.test.ts

```diff
@@ -34,7 +35,9 @@ test('returns config at path as observable', async () => {
   const configService = new ConfigService(config$, defaultEnv, logger);
 
   const configs = configService.atPath('key', ExampleClassWithStringSchema);
```

Comment:
How would you feel about tweaking the name of `configs` to `config$` since it's a config stream? I realize TypeScript takes care of that information for us, so it's not super important, but I would still find it easier to read PRs on Github if the observable variables used the `$` postfix.